src/vs/workbench/parts/debug/electron-browser/variablesView.ts

```diff
@@ -179,6 +179,7 @@ class VariablesActionProvider implements IActionProvider {
 		const actions: IAction[] = [];
 		const variable = <Variable>element;
 		actions.push(new SetValueAction(SetValueAction.ID, SetValueAction.LABEL, variable, this.debugService, this.keybindingService));
+		actions.push(new CopyPropertyPathAction(CopyPropertyPathAction.ID, CopyPropertyPathAction.LABEL, variable));
```

Comment:
Set Value and Copy Value should be first, So CopyEvaluatePathAction should come third